src/System.Threading.Thread/tests/UnknownMain/UnknownMain.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+
+namespace Unknown
+{
+    internal static class Program
+    {
+        static int Main(string[] args)
+        {
+            int retValue = 0;
+
+            int mode = int.Parse(args[0]);
+            if (mode == 1)
```

Comment:
I think a string would be clearer than an int for the mode because then I don't have to check both sides to see it's correct